spark/src/main/java/org/apache/iceberg/actions/Actions.java

```diff
@@ -77,6 +82,81 @@ public ExpireSnapshotsAction expireSnapshots() {
     return new ExpireSnapshotsAction(spark, table);
   }
 
+  /**
+   * Converts the provided table into an Iceberg table in place. The table will no longer be represented by it's
+   * previous provider in the session catalog and a new metadata directory will be created at the table's location.
```

Comment:
The doc here is really specific to Spark, but I don't think there is a need for it. How about something like "The table will no longer be accessible using the previous implementation"?

Also, new data will be written to the `data` directory to avoid breaking any copies of the table. When we migrate, we leave a `db.table_hive` copy that can be renamed back in place to roll back the operation. The user is responsible for dropping the `_hive` copy or doing renames to rollback. Not sure how much of that applies to this PR though.

Reply:
I'll consider the fall back behavior in the action itself, I think we could leave another copy behind but I think that probably should be a user operation rather than a automatic part of migrate. I could be convinced otherwise though :shrug:

Reply:
This seems to be still open.

Reply:
woops only converted the lower one, this one is fixed too now